static/images/jetpack/jetpack-connection-bad.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="151" height="151" viewBox="0 0 151 151" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
By using a tool like https://jakearchibald.github.io/svgomg/  we can make his svg even smaller.